gnocchi/opts.py

```diff
@@ -218,6 +218,32 @@ def list_opts():
                 default=10,
                 help='Delay between flushes'),
         )),
+        ("amqp1d", (
+            cfg.HostAddressOpt('host',
+                               default='localhost',
+                               required=True,
+                               help='The listener IP for amqp1.0 '),
+            cfg.PortOpt('port',
+                        default=5672,
+                        help='The port for amqp 1.0'),
+            cfg.StrOpt('topic',
+                       default='u/collectd/telemetry',
+                       help='Topic name to listen to for  1.0'),
+            cfg.StrOpt('data_source',
+                       default='collectd',
+                       choices=['collectd'],
+                       help='Data source for amqp1d(currently supporting '
+                       ' only collectd)'),
```

Comment:
missing space in front of (

no need to say only collectd is supported, that's written by the generator.

Reply:
done